docs/TRY_IT_OUT_EXECUTOR.md

```diff
@@ -0,0 +1,219 @@
+#### TryItOut Executor
+
+TryItOut Executor is an OpenAPI specific HTTP client for OAS operations.
+It maps an OAS operation and values into an HTTP request.
+
+To execute an OAS operation, you need to provide TryItOut Executor with various properties.
+
+Type notations are formatted like so:
+
+- `String=""` means a String type with a default value of `""`.
+- `String=["a"*, "b", "c", "d"]` means a String type that can be `a, b, c, or d`, with the `*` indicating that `a` is the default value.
+
+Property | Description
+--- | ---
+`spec` | `Object`, **REQUIRED**. OpenAPI definition represented as [POJO](https://en.wikipedia.org/wiki/Plain_old_Java_object).
+`operationId` | `String`. Unique string used to identify an operation. If not provided, `pathName` + `method` must be used instead.
+`pathName` | `String`. OpenAPI defines a unique operation as a combination of a path and an HTTP method. If `operationId` is not provided, this property must be set.
+`method` | `String=["GET", "HEAD", "POST", "PUT", "DELETE", "CONNECT", "OPTIONS", "TRACE", "PATCH"]`. OpenAPI defines a unique operation as a combination of a path and an HTTP method. If `operationId` is not provided, this property must be set.
+`parameters` | `Object`. Parameters object, eg: `{ q: 'search string' }`. Not recognized parameters (not defined in `spec`) will be ignored.
```

Comment:
"unrecognized" and "unsupported" parameters

Reply:
"unsupported" doesn't seem to be appropriate. What I'm trying to express here, is that all `parameters` that are not defined in original `spec` will not be used in request. I'll try to use more descriptive language. thanks